cms/templates/js/video-transcripts.underscore

```diff
@@ -0,0 +1,49 @@
+<div class='show-video-transcripts-container'>
+<% if (_.size(transcripts)) { %>
+    <button class="button-link toggle-show-transcripts-button">
+        <strong>
+        <span class="icon fa fa-caret-right toggle-show-transcripts-icon" aria-hidden="true"></span>
+        <span class="toggle-show-transcripts-button-text">
+            <%- StringUtils.interpolate(gettext('Show transcripts ({totalTranscripts})'), {totalTranscripts: _.size(transcripts)})%>
+        </span>
+        </strong>
+    </button>
+<% } else { %>
+    <span class='transcripts-empty-text'><%- gettext('No transcript available yet.') %></span>
+<% }%>
+<div class='show-video-transcripts-wrapper hidden'>
+<% _.each(transcripts, function(transcriptLanguageText, transcriptLanguageCode){ %>
+    <% selectedLanguageCodes = _.keys(_.omit(transcripts, transcriptLanguageCode)); %>
+    <div class='show-video-transcript-content' data-edx-video-id="<%- edxVideoID %>" data-language-code="<%- transcriptLanguageCode %>">
+        <div class='transcript-upload-status-container'></div>
+        <strong class='transcript-title'><%- StringUtils.interpolate(gettext('{transcriptClientTitle}_{transcriptLanguageCode}.{fileExtension}'), {transcriptClientTitle: transcriptClientTitle, transcriptLanguageCode: transcriptLanguageCode, fileExtension: transcriptFileFormat}) %></strong>
+        <select class='transcript-language-menu'>
+            <option value=''>Select Language</option>
+            <% _.each(transcriptAvailableLanguages, function(availableLanguage){ %>
+                <% if (!_.contains(selectedLanguageCodes, availableLanguage[1])) { %>
+                    <option value='<%- availableLanguage[1] %>' <%- transcriptLanguageCode === availableLanguage[1] ? 'selected': '' %>><%- availableLanguage[0] %></option>
+                <% } %>
+            <% }) %>
+        </select>
+        <input class="upload-transcript-input hidden" type="file" name="file" accept=".<%- transcriptFileFormat %>"/>
+        <div class='transcript-actions'>
+            <a
+                class="button-link download-transcript-button"
+                href="<%- StringUtils.interpolate(
+                    '{transcriptDownloadHandlerUrl}?edx_video_id={edxVideoID}&language_code={transcriptLanguageCode}',
+                    {
+                        transcriptDownloadHandlerUrl: transcriptDownloadHandlerUrl,
+                        edxVideoID: edxVideoID,
+                        transcriptLanguageCode: transcriptLanguageCode
+                    }
+                ) %>"
+            >
+                <%- gettext('Download') %>
+            </a>
+            <span class='transcript-actions-separator'> | </span>
+            <button class="button-link upload-transcript-button"><%- gettext('Upload') %></button>
```

Comment:
User facing upload button string

Reply:
👍 

Though when I tried this, the system let me choose an .srt file for upload, but the file never showed up on the page. Is this button supposed to allow the user to replace the transcript that's already there?

Reply:
Yes this UX is to replace a transcript already uploaded. Do you think we should change this button to `Replace` ? @sstack22 your thoughts?

Eventually there will be another button that will let user upload/add a new transcript (but that's future work).

Reply:
Yeah, I think "Replace" would be a lot less confusing. @sstack22 ?

I'm still not seeing the actual replacement behavior, though—when I upload a new file, I see a green "Transcript uploaded" message, but the name of the old file remains.

Reply:
@srpearce See the bold transcript name eg: **test2_am.srt** after being replaced with English name changes to **test2_en.srt**

Reply:
@sstack22 Thoughts on "Replace" text ?